test/Microsoft.Sbom.Common.Tests/CommonSPDXUtilsTests.cs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Parser;
+
+using System.Data;
+using Microsoft.Sbom.Common.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+[TestClass]
+public class CommonSPDXUtilsTests
+{
+    private const string TestPackage1 = "testPackage1";
+    private const string TestPackage2 = "testPackage2";
+    private const string TestPackage1SpdxId = "SPDXRef-Package-17BE63B07D94E1FDAFCC6AD9F7DB98CC90A7A17744E94C045A10FE15DD426FFE";
+    private const string TestPackage2SpdxId = "SPDXRef-Package-B7A5E408E59C8D95B09020EA79327C05272A8EE8C6B11331775A004AA0EED04D";
+
+    private const string TestFile1 = "testFile1";
+    private const string TestFile2 = "testFile2";
+
+    private const string TestExternalDoc1 = "testExternalDoc1";
+    private const string TestExternalDoc2 = "testExternalDoc2";
+
+    private const string TestHash1 = "sha1Value1";
+    private const string TestHash2 = "sha1Value2";
+
+    private const string TestFile1Hash1SpdxId = "SPDXRef-File-testFile1-sha1Value1";
+    private const string TestFile2Hash2SpdxId = "SPDXRef-File-testFile2-sha1Value2";
+    private const string TestExternalDoc1Hash1SpdxId = "DocumentRef-testExternalDoc1-sha1Value1";
+    private const string TestExternalDoc2Hash2SpdxId = "DocumentRef-testExternalDoc2-sha1Value2";
+
+    [DataRow(TestPackage1, TestPackage1SpdxId)]
+    [DataRow(TestPackage2, TestPackage2SpdxId)]
+    [TestMethod]
+    public void GenerateSpdxPackageIdTest(string package, string expectedSpdxId)
+    {
+        var spdxId = CommonSPDXUtils.GenerateSpdxPackageId(package);
+        Assert.AreEqual(expectedSpdxId, spdxId);
+    }
+
+    [TestMethod]
+    public void GenerateSpdxPackageIdTest_SameStrings()
+    {
+        var spdxId1 = CommonSPDXUtils.GenerateSpdxPackageId(TestPackage1);
+        var spdxId2 = CommonSPDXUtils.GenerateSpdxPackageId(TestPackage1);
+        Assert.AreEqual(spdxId1, spdxId2);
+    }
+
+    [TestMethod]
+    public void GenerateSpdxPackageIdTest_DiffStrings()
```

Comment:
Again, maybe not needed since we're comparing to const strings? I suppose we could add a test that the constant strings are different, just in case someone ever decided to edit them, but that could be a bit of a stretch